firebase-firestore/src/test/java/com/google/firebase/firestore/local/LocalStoreTestCase.java

```diff
@@ -159,7 +159,7 @@ private void acknowledgeMutation(long documentVersion, @Nullable Object transfor
             version,
             transformResult != null
                 ? Collections.singletonList(TestUtil.wrap(transformResult))
-                : null);
+                : Collections.emptyList());
```

Comment:
optional nit: you can just remove the Nullable annotation in the signature and change this method's overload to pass in an empty list instead of `null`.

Reply:
Hm, this would have been nice but unfortunately `wrap` throws when it is passed an empty Object.